src/reporting/market.se

```diff
@@ -359,6 +351,24 @@ def validatePayoutNumerators(payoutNumerators: arr):
     require(sum == self.payoutDenominator)
     return 1
 
+#CONSIDER: Do these 2 functions need any other require()s?
+def transferBondAmountToDisputeBondToken(bondHolder: address, disputeBondToken: address, bondAmount: int256):
+    reputationToken = self.getReputationToken()
```

Comment:
I think it might be reasonable to `assertPrivate` and require that the bondHolder is actually the bondHolder of the disputeBondToken